custom/abdm/milestone_one/utils/abha_creation_util.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from custom.abdm.milestone_one.utils.request_util import get_response_http_post
+
+logger = logging.getLogger(__name__)
+
+REG_API_URL = "v1/registration/aadhaar/"
+GENERATE_AADHAAR_OTP_URL = REG_API_URL + "generateOtp"
+GENERATE_MOBILE_OTP_URL = REG_API_URL + "generateMobileOTP"
+VERIFY_AADHAAR_OTP_URL = REG_API_URL + "verifyOTP"
+VERIFY_MOBILE_OTP_URL = REG_API_URL + "verifyMobileOTP"
+CREATE_HEALTH_ID_URL = REG_API_URL + "createHealthIdWithPreVerified"
+
+
+def generate_aadhar_otp(aadhaar_number):
```

Comment:
nit: `aadhaar` and not `aadhar`

Reply:
https://github.com/dimagi/commcare-hq/pull/32332/commits/0b755dd44470292cbeda6c6ed7e11657449b86b5